src/ui/templates/insurance/export-contract/how-was-the-contract-awarded.njk

```diff
@@ -16,19 +18,111 @@
     }
   }) }}
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-three-quarters-from-desktop">
-      <div class="govuk-grid-column-three-quarters-from-desktop govuk-!-padding-right-0 govuk-!-padding-left-0">
-        <span class="govuk-caption-l" data-cy="heading-caption">{{ CONTENT_STRINGS.HEADING_CAPTION }}</span>
-        <h1 class="govuk-heading-l" data-cy="{{ DATA_CY.HEADING }}">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
-      </div>
-    </div>
-  </div>
+  <span class="govuk-caption-xl" data-cy="heading-caption">
+    {{ CONTENT_STRINGS.HEADING_CAPTION }}
+  </span>
+
+  {% set otherMethodHtml %}
+    {% include CONDITIONAL_OTHER_METHOD_HTML %}
+  {% endset %}
 
   <form method="POST" data-cy="form" enctype="application/x-www-form-urlencoded" novalidate>
 
     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
 
+    {{ govukRadios({
```

Comment:
could this be in its own component/partial and imported in?

Reply:
Yes, but I don't see much value in it, as it's specific to this pagea and won't be used anywhere else. Think it would overcomplicate it?

Reply:
just thinking since the file is pretty big with this in, to make it slightly more readable